examples/tools/tool-package-quickstart/my_tool_package/yamls/my_tool_package.tools.mlindex_tool.yml

```diff
@@ -0,0 +1,49 @@
+my_tool_package.tools.mlindex_tool.my_tool:
+  function: my_tool
+  inputs:
+    index_json:
+      type:
+      - string
+      generated_by_ux:
```

Comment:
This is only for user-friendly in UX.
1. In SDK customer needs to put the raw json
2. In UX customer can put the raw json or use something like popup window to fill the parameters, UX will generate the index json by calling func （generate_index_json）provided by customer